app/models/lab.rb

```diff
@@ -1,3 +1,10 @@
 class Lab < ApplicationRecord
   has_many :scientists
-end
\ No newline at end of file
+
+  def self.scientists_with_experiments
+    joins(scientists: :scientist_experiments)
+      .select("scientists.name, scientist_experiments.scientist_id AS sci_id, COUNT(scientist_experiments.scientist_id) AS experiment_count")
+      .group("scientists.name, scientist_experiments.scientist_id")
+      .order("experiment_count DESC")
+  end
```

Comment:
This AR is not correct for what you are trying to do. We only want Scientists who work at the particular lab, not all the scientists across all Labs.